pkg/server/node.go

```diff
@@ -422,7 +423,10 @@ func (s *nodeServer) getFetchBaseSVIDResponse(
 
 func (s *nodeServer) getSpiffeIDFromCtx(ctx context.Context) (spiffeID string, err error) {
 
-	ctxPeer, _ := peer.FromContext(ctx)
+	ctxPeer, ok := peer.FromContext(ctx)
+	if !ok {
+		return "", errors.New("It was not posible to read a SVID from your request")
```

Comment:
I have been thinking about this lately, and thought that we need stricter enforcement around client certificate usage given various endpoints since we request but don't require one.... that said, perhaps this error is enough to get that across. suggest: `"an SVID is required for this request"`